mk/spksrc.checksum.mk

```diff
@@ -39,18 +39,13 @@ pre_checksum_target: checksum_msg
 
 # validate file integrity with the provided digests.
 checksum_target: $(PRE_CHECKSUM_TARGET)
-	@if [ -f $(DIGESTS_FILE) ] ; \
-	then \
-	  cat $(DIGESTS_FILE) | (cd $(DISTRIB_DIR) && while read file type value ; \
+	@if [ -f $(DIGESTS_FILE) ] ; then \
```

Comment:
I had a much simpler version to manage more than one file in the `digests`.  See https://github.com/SynoCommunity/spksrc/commit/560e379d9eb7134a14af0e72e72013683372958f
I'm not sure the added complexity is needed but I don't fully get what is tried to be accomplished here.